internal/devconfig/configfile/env.go

```diff
@@ -25,10 +26,10 @@ func (c *ConfigFile) ParseEnvsFromDotEnv() (map[string]string, error) {
 	if !c.IsdotEnvEnabled() {
 		return nil, fmt.Errorf("env file does not have a .env extension")
 	}
-
-	file, err := os.Open(c.EnvFrom)
+	envFileAbsPath := path.Join(filepath.Dir(c.AbsRootPath), c.EnvFrom)
```

Comment:
This breaks with absolute paths. I think we need to treat absolute and relative paths differently.

Reply:
pushed a commit to handle this case.

Reply:
What you want is:

```golang
envfileAbsPath := c.EnvFrom
if !filepath.IsAbs(c.EnvFrom) {
  envfileAbsPath = filepath.Join(c.AbsRootPath, c.EnvFrom)
}
```